addons/pos_self_order/static/src/mobile/pages/product_main_view/product_main_view.js

```diff
@@ -54,9 +75,21 @@ export class ProductMainView extends Component {
         }));
         return { ...attribute, values };
     }
+    updateMultipleAttribute(event, attributeId) {
+        const value = parseInt(event.target.id);
 
+        if (this.state.selectedVariants[attributeId].has(value)) {
+            this.state.selectedVariants[attributeId].delete(value);
+        } else {
+            this.state.selectedVariants[attributeId].add(value);
+        }
+    }
```

Comment:
again, this should be a `t-model`. Also, it is weird to always have every single thing from kiosk duplicated in self and the other way around. It's clear that `kiosk` and `self` are different things, but still, they share most of the functionality, but almost none of the code.